.github/workflows/dependency-review.yml

```diff
@@ -1,20 +1,15 @@
-# Dependency Review Action
-#
-# This Action will scan dependency manifest files that change as part of a Pull Request, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
-#
-# Source repository: https://github.com/actions/dependency-review-action
-# Public documentation: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
 name: 'Dependency Review'
-on: [pull_request]
+on:
+    pull_request:
+        branches: ['master']
 
 permissions:
     contents: read
+    pull-requests: write
```

Comment:
What is this for?

Reply:
Good question.
https://github.com/actions/dependency-review-action
This is to have a report on PR when something goes wrong